tests/func/test_remote.py

```diff
@@ -64,7 +47,9 @@ def test_overwrite(self):
         remote_name = "a"
         remote_url = "s3://bucket/name"
         self.assertEqual(main(["remote", "add", remote_name, remote_url]), 0)
-        self.assertEqual(main(["remote", "add", remote_name, remote_url]), 1)
+        self.assertNotEqual(
```

Comment:
is it still better to check an exact return value? it can be considered as a part of the protocol sometimes - different non 0 return values for different values

Reply:
It needs to be revisited. Added https://github.com/iterative/dvc/issues/2087 .